server/app.ts

```diff
@@ -115,32 +119,13 @@ Created default user
 	Password: ${DEFAULT_PASSWORD}
 **Delete this user after you have used it to set up your account**
 	`);
-})();
 
-let authenticateWithReject = async function (request: express.Request, response: express.Response, next: express.NextFunction) {
-	let authKey = request.cookies.auth;
-	let user = await User.findOne({"auth_keys": authKey});
-	if (!user) {
-		response.status(401).json({
-			"error": "You must log in to access this endpoint"
-		});
-	}
-	else {
-		response.locals.username = user.username;
-		next();
-	}
-};
-let authenticateWithRedirect = async function (request: express.Request, response: express.Response, next: express.NextFunction) {
-	let authKey = request.cookies.auth;
-	let user = await User.findOne({"auth_keys": authKey});
-	if (!user) {
-		response.redirect("/login");
-	}
-	else {
-		response.locals.username = user.username;
-		next();
-	}
-};
+	// Add default list of tags 
+	let defaultTagsList = new TagsList({
+		tags: ["hackgt", "breakfast", "lunch", "dinner", "visited-microsoft"]
```

Comment:
Are we sure we want these to be the default tags instead of something like an empty array or just `hackgt`?

Reply:
True, always having these as the defaults doesn't really make sense. We can just start with `hackgt` as the default tag.